src/main/java/org/jenkinsci/bytecode/Transformer.java

```diff
@@ -57,16 +58,28 @@ public synchronized void loadRules(Collection<? extends ClassLoader> loaders) th
             return image;
 
         final ClassReader cr = new ClassReader(image);
-        final ClassWriter cw = new ClassWriter(/*ClassWriter.COMPUTE_FRAMES|*/ClassWriter.COMPUTE_MAXS);
+        final ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_FRAMES|ClassWriter.COMPUTE_MAXS);
```

Comment:
it was never removed.  It was entered with it commented out in the first revision...